### PR TITLE
[4.4] Run ipa-custodia under Python 2

### PR DESCRIPTION
**Pull request by tiran**
Closes: https://pagure.io/freeipa/issue/6926
Signed-off-by: Christian Heimes <cheimes@redhat.com>

### CODE REVIEW

File: init/systemd/ipa-custodia.service [hunk 1]
```diff
@@ -3,8 +3,7 @@ Description=IPA Custodia Service
 
 [Service]
 Type=simple
-
-ExecStart=/usr/sbin/custodia /etc/ipa/custodia/custodia.conf
+ExecStart=@libexecdir@/ipa/ipa-custodia /etc/ipa/custodia/custodia.conf
```
Comment (stlaz) [inline init/systemd/ipa-custodia.service]: This macro does not get replaced resulting in the service not being properly configured.

**Comment by stlaz** [comment]
Works for me. However, I do not see the reason to do `custodia > 0.2`, please, either provide some or remove it.

**Comment by tiran** [comment]
The patch doesn't work with custodia 0.1. It needs at least 0.2.0-1.

**Comment by stlaz** [comment]
Alright, thanks. ACK.

**Comment by tomaskrizek** [comment]
@tiran Once this PR is merged, do you plan to remove the conflict with `freeipa-server-common < 4.5` from custodia or do we need to address something else as well?

**Comment by tiran** [comment]
@tomaskrizek yes, custodia 0.5 will no longer have the restriction.

**Comment by tomaskrizek** [comment]
Needs re-base for 4.4 and 4.5.

**Comment by tomaskrizek** [comment]
ipa-4-4:

* 307c4bd62609c9ac58633e3ccc61d85e2caacbcc Run ipa-custodia under Python 2

**Comment by lslebodn** [comment]
@tiran Do you plan to release custodia-0.2 in fedora 25?
Because this patch broke building of ipa-4.4 on fedora 25.

If you do not plan to upgrade custodia in f25 then this patch should be (partially) reverted

**Pull request closed without merging**